networking/src/Node/Internal.hs

```diff
@@ -64,22 +70,39 @@ import           Data.NonEmptySet (NonEmptySet)
 import qualified Data.NonEmptySet as NESet
 import           Data.Set (Set)
 import qualified Data.Set as Set
+import           Data.Text (Text)
+import           Data.Time.Clock.POSIX (getPOSIXTime)
 import           Data.Time.Units (Microsecond)
 import           Formatting (sformat, shown, (%))
 import           GHC.Generics (Generic)
-import qualified Mockable.Channel as Channel
-import           Mockable.Class
-import           Mockable.Concurrent
-import           Mockable.CurrentTime (CurrentTime, currentTime)
-import qualified Mockable.Metrics as Metrics
-import           Mockable.SharedAtomic
-import           Mockable.SharedExclusive
-import qualified Network.Transport as NT (EventErrorCode (..))
-import qualified Network.Transport.Abstract as NT
+import qualified Network.Transport as NT
 import           Node.Message.Class (Packing, Serializable (..), pack, unpack)
 import           Node.Message.Decoder (Decoder (..), DecoderStep (..), continueDecoding)
+import qualified System.Metrics.Distribution as Metrics (Distribution)
+import qualified System.Metrics.Gauge as Metrics (Gauge)
+import qualified System.Metrics.Distribution as Metrics.Distribution
+import qualified System.Metrics.Gauge as Metrics.Gauge
 import           System.Random (Random, StdGen, random)
-import           System.Wlog (WithLogger, logDebug, logError, logWarning)
+import qualified System.Wlog as Wlog (usingLoggerName, logInfo, logDebug, logError, logWarning)
+
+-- Copied from the old Mockable definition for Production.
+getCurrentTime :: IO Microsecond
+getCurrentTime = round . (* 1000000) <$> getPOSIXTime
+
+delay :: Microsecond -> IO ()
+delay = threadDelay . fromIntegral
+
+logInfo :: Text -> IO ()
+logInfo = Wlog.usingLoggerName "node" . Wlog.logInfo
+
+logDebug :: Text -> IO ()
+logDebug = Wlog.usingLoggerName "node" . Wlog.logDebug
+
+logWarning :: Text -> IO ()
+logWarning = Wlog.usingLoggerName "node" . Wlog.logWarning
+
+logError :: Text -> IO ()
+logError = Wlog.usingLoggerName "node" . Wlog.logError
```

Comment:
Oops, this was done before I decided on using what's in `Pos.Util.Trace`. I can knock these out and use a `Trace` instead.

Reply:
Yes, please do :)